tools/gn

```diff
@@ -308,6 +308,12 @@ def to_gn_args(args):
       gn_args['skia_use_metal'] = True
       gn_args['shell_enable_metal'] = True
 
+    # Enable Vulkan on all platforms.
+    gn_args['skia_use_vulkan'] = True
+    gn_args['shell_enable_vulkan'] = True
```

Comment:
I believe this will compile the vulkan shell component even on iOS and Android. The desire is to compile Vulkan on all host unit-test targets right?

Reply:
My intention was to enable Vulkan support for all libflutter platform distributions including iOS, so that embedder writers have the option to target just Vulkan and use translation ICDs like MoltenVK. Partially because I want it, but also just because we can. :)

Reply:
With that, you run into the issue of bumping up the binary size on those target. And we are pretty sensitive to that metric.

Reply:
I'll disable this for Android/iOS for the time being and leave a note since it's easier to flip on than flip off later.